### DISCUSSION TITLE
Replace deployment in heroku with other platform

### DISCUSSION THREAD

**Discussion by Rakesh7475** [community/community#46366] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Bro all the app you have deployed are in heroku app. 
can you please deploy it in another app. it will be good. As heroku is not free now. 
they have started charging now

**Comment by Anmol-Baranwal** [comment]
You can use several platforms like
- render (use this to deploy backend projects)
- netlify
- vercel
- firebase
- GitHub Pages
- render
- AWS
- Azure

and many more to deploy according to your application needs

**Reply by Anmol-Baranwal** [reply]
@Rakesh7475 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
